demos/python_word_count/word_count.py

```diff
@@ -4,27 +4,26 @@
 
 
 def application_setup(args):
-    print("Word Count")
     in_host, in_port = wallaroo.tcp_parse_input_addrs(args)[0]
     out_host, out_port = wallaroo.tcp_parse_output_addrs(args)[0]
 
     lines = wallaroo.source("Split and Count",
-                        wallaroo.TCPSourceConfig(in_host, in_port, decoder))
+                        wallaroo.TCPSourceConfig(in_host, in_port, decode_lines))
```

Comment:
`decode_lines` should be `decode_line`, right?

Reply:
The way word count is written right now, the inputs are strings that might contain multiple lines (this is why `split` first splits on `"\n"`).  So abstractly we're decoding multiple lines per input.  However, in fact, we're decoding a single string per input.  I wasn't sure which way to go with the naming here.